docs/src/main/docs/guides/01_SE_REST_web-service.adoc

```diff
@@ -20,7 +20,7 @@
 :example-src-url: https://github.com/oracle/helidon/tree/master/examples/guides/se-rest-web-service/src/main/java
 
 
-= Creating a ReSTful Web Service using Helidon SE
+= Helidon SE RESTful Web Service
```

Comment:
I suggest: "Create a RESTful Web Service Using Helidon SE"

The "Create" verb is a useful signpost to help users know in advance what content they can expect in the doc. They'll know that the doc contains a step-by-step procedure, as opposed to a conceptual outline, for example.

PS: In formal Oracle docs, the gerund form "Creating.." used to be the norm for titles. But that's changing now to the simpler direct verb form.

Reply:
the issue we have is the left nav panel on the page is so narrow that the "SE" part and the MP part for the corresponding label for the MP guide are obscured. The user does not see a visible difference (although the tool tips do show the entire label).

We could try "Create Helidon SE RESTful Web Service" although stringing together so many nouns consecutively can become less clear - but here brevity is in order!

Reply:
I like that a lot. I'll make that change for both guides and push those changes.

Reply:
I made those changes locally and the nav panel shows: 

Helidon SE: Create a RESTful Web...
Helidon MP: Create a RESTful Web...

but that's probably as good as it gets. I'll push those changes.